bvm/ballerina-core/src/main/java/org/ballerinalang/model/types/BTypes.java

```diff
@@ -111,4 +111,14 @@ public static BType fromString(String typeName) {
         }
         return getTypeFromName(typeName);
     }
+
+    public static boolean isImplicitInitValueNil(BType type) {
```

Comment:
this is equivalent to checking nillable right? so can we call it `isNillable()` and make it generic?

Reply:
This is in the runtime. The `isNullable()` method is there only in the compile time `BType`. Will have to add it to the runtime `BType` as well